### DISCUSSION TITLE
Toggle fields

### DISCUSSION THREAD

**Discussion by timsofteng** [react-hook-form/react-hook-form#9926] [General]
Hello.
I have toggle near the input field which should keep value in field but disable its validation prevent input's value from be pushed with rest data in submit. Next pressing on toggle should revert it into common form input.

What is the best way to do it?

**Comment by bluebill1049** [comment]
use `watch` API?

**Reply by timsofteng** [reply]
> use `watch` API?

May you describe please how can it help?

**Comment by timsofteng** [comment]
@bluebill1049 still cannot solve this issue. may you explain please what did you mean?
